rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -2023,102 +1919,54 @@ def test_add_ksm_blockchain_account_invalid_ens_domain(rotkehlchen_api_server):
 @pytest.mark.parametrize('kusama_manager_connect_at_start', [KUSAMA_TEST_NODES])
```

Comment:
You marked this as resolved but isn't it removable? Won't it now connect to kusama node the first time you add a kusama account?

Reply:
Nope, as I found out it is not removable. This sets kusama manager's `connect_at_start` variable which contains a list of nodes and used when we call `attempt_connections` when adding an account. And if we don't set `kusama_manager_connect_at_start`, there will basically be no nodes to connect to.